doc/bookkeeper-options.md

```diff
@@ -78,3 +78,19 @@ Default gcLoggingOpts:
 "-XX:NumberOfGCLogFiles=5",
 "-XX:GCLogFileSize=64m",
 ```
+
+### BookKeeper Custom Configuration
+
```

Comment:
This description does not look right. 
We need to mention that when using BK Cluster for Pravega it is necessary to create a config map with these values:
```
PRAVEGA_CLUSTER_NAME --> name of Pravega Cluster using this BK
WAIT_FOR ---> Zookeeper URL
```
Its important to explain what those values stand for.
How he creates the config map is something we don't care about.
As mentioned over slack we need to make changes to bookkeeper charts and manual deployment files to add yaml for the user config map.